discovery-infra/tests/conftest.py

```diff
@@ -54,6 +54,7 @@ def _get_cluster_name():
                  "cluster_name": _get_cluster_name(),
                  "private_ssh_key_path": utils.get_env('PRIVATE_KEY_PATH', private_ssh_key_path_default),
                  "kubeconfig_path": utils.get_env('KUBECONFIG', ''),
+                 "installer_kubeconfig_path": utils.get_env('INSTALLER_KUBECONFIG', None),
```

Comment:
we need to add ``INSTALLER_KUBECONFIG`` to ``skipper.env``, no?
also giving it a default value on the ``Makefile``

Reply:
added to skipper.env 
not sure we need to add it to Makefile, if not set it will go to the default path